Dockerfile

```diff
@@ -10,8 +10,17 @@ RUN { \
 WORKDIR /usr/src/app
 COPY . .
 
+RUN rm -r Gemfile.lock .ruby-lsp .git .github .DS_Store
```

Comment:
I've also done this in projects before where I use the `.dockerignore` file as a whitelist when there's only a handful of files/directories I want to include into the build context. Similar to gitignore you can "unignore" paths with `!<path>` lines, so something like

```
.*
*
!/bin/
!/lib/
!/test/
!/Gemfile
!/Rakefile
!/oas_agent.gemspec
```

Would load just those paths into docker regardless of what the local folder contains otherwise. (`tmp/`, etc.)